web/frontend/package.json

```diff
@@ -22,11 +22,16 @@
   "dependencies": {
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
+    "axios": "^0.24.0",
+    "formik": "^2.2.9",
```

Comment:
Same question: why we need this dependency?

Reply:
Didn't know how to do form validation, used what was in the [template](https://material-ui.com/store/items/devias-kit/)
Redesigned validation, removed formik